src/components/LoanCards/Buttons/CheckoutNow.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<kv-button
+		class="success"
+		v-kv-track-event="['Lending', 'click-Read more', 'checkout-now-click', id, 'true']">
+		<kv-icon name="check-in-circle" />
+		Checkout now
+	</kv-button>
+</template>
+
+<script>
+import KvIcon from '@/components/Kv/KvIcon';
+import KvButton from '@/components/Kv/KvButton';
+
+export default {
+	components: {
+		KvIcon,
+		KvButton,
+	},
+	props: {
+		id: {
+			type: Number,
+			default: null
+		},
+	},
+};
+
+</script>
+
+<style lang="scss">
+@import 'settings';
+
+.success.button {
```

Comment:
The styles for this `.success` class look pretty similar to the `.secondary` class styles that KvButton already defines. Could that be used instead? If they are different enough then maybe these should be added into the KvButton component so that they can be used elsewhere.

Reply:
I'll look into this, thanks for the review @emuvente